src/NugetReferenceResolver/RuntimeGraph.cs

```diff
@@ -0,0 +1,926 @@
+﻿using Microsoft.DotNet.PlatformAbstractions;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace NugetReferenceResolver
+{
+    public class RuntimeGraph
+    {
+        private const string RuntimeCompatibility = @"
```

Comment:
Ok. You might consider this an embedded resource so it will be easier to update in the future. At the very least, move this into a separate file `partial class` so it's easier to see that there is also code in this file, not just data.